docs/updating/8-0.md

```diff
@@ -0,0 +1,182 @@
+
+While migrating to include the new dark theme files is unlikely to cause breaking changes, these new selectors can lead to unexpected overrides if custom CSS variables are being set on the `body` element. We recommend updating any instances where global application variables are set to target the `:root` selector instead.
+
+For more information on the new dark theme files, refer to the [Dark Mode documentation](../theming/dark-mode).
+
+### Dynamic Font
+
+The `core.css` file has been updated to enable dynamic font scaling by default.
+
+The `--ion-default-dynamic-font` variable has been removed and replaced with `--ion-dynamic-font`.
+
+Developers who had previously chosen dynamic font scaling by activating it in their global stylesheets can revert to the default setting by removing their custom CSS. In doing so, their application will seamlessly continue utilizing dynamic font scaling as it did before. It's essential to note that altering the font-size of the html element should be avoided, as it may disrupt the proper functioning of dynamic font scaling.
+
+Developers who want to disable dynamic font scaling can set `--ion-dynamic-font: initial;` in their global stylesheets. However, this is not recommended because it may introduce accessibility challenges for users who depend on enlarged font sizes.
+
+For more information on the dynamic font, refer to the [Dynamic Font Scaling documentation](../layout/dynamic-font-scaling).
+
+## Required Changes
+
+### Browser Support
+
+The list of browsers that Ionic supports has changed. Review the [Browser Support Guide](../reference/browser-support) to ensure you are deploying apps to supported browsers.
+
+If you have a `browserslist` or `.browserslistrc` file, update it with the following content:
+
+```
+Chrome >=89
+ChromeAndroid >=89
+Firefox >=75
+Edge >=89
+Safari >=15
+iOS >=15
+```
+
+### Checkbox
+
+1. Migrate any remaining instances of Checkbox to use the [modern form control syntax](../api/checkbox#migrating-from-legacy-checkbox-syntax). Additionally, remove any usages of the `legacy` property as the legacy form control syntax has been removed.
+
+### Input
+
+1. Remove any usages of the `size` property. CSS should be used to specify the visible width of the input instead.
+2. Remove any usages of the `accept` property.
+3. Migrate any remaining instances of Input to use the [modern form control syntax](../api/input#migrating-from-legacy-input-syntax). Additionally, remove any usages of the `legacy` property as the legacy form control syntax has been removed.
+
+### Nav
+
+1. Update any usages of `getLength` to `await` the call before accessing the returned value as this method now returns `Promise<number>` instead of `number`.
+
+### Picker
+
+1. Ionic 8 now ships with an inline `ion-picker` component. Developers who wish to continue using the legacy picker should update any `ion-picker` usages to `ion-picker-legacy`. The `pickerController` import remains unchanged.
```

Comment:
Should we mention that `ion-picker-legacy` will be removed in the next major release?

Reply:
Good call. Added in https://github.com/ionic-team/ionic-docs/pull/3475/commits/7bdf9f2359c2406e0a85ce6eab672747093e0bf0